arkworks/src/kzg_types.rs

```diff
@@ -73,31 +76,43 @@ fn bigint_check_mod_256(a: &[u64; 4]) -> bool {
     overflow
 }
 
+#[repr(C)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
+pub struct ArkFr(pub blst_fr);
+
+impl Default for LFFTSettings {
+    fn default() -> Self {
+        Self::new(0).unwrap()
+    }
+}
+
```

Comment:
These (and all other structs) should not change: type `ArkFr` should wrap field element from `arkworks` crate, not blst.